webapp/pages/profile/_id/_slug.vue

```diff
@@ -327,6 +334,11 @@ export default {
     },
   },
   methods: {
+    handleTab(tab) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-06-05T20:52:47Z" title="Wednesday, June 5th 2019, 10:52:47 pm +02:00">Jun 5, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -317,6 +321,19 @@ export default {
     },
   },
   methods: {
+    handleTab(tab) {
```


```suggestion
    handleShoutedTab() {},
    handleCommentedTab() {},
    handlePostTab() {}


```